spark/processing/3.0/py3/hadoop-config/hdfs-site.xml

```diff
@@ -3,17 +3,60 @@
  <!-- Put site-specific property overrides in this file. -->
 
  <configuration>
-     <property>
-         <name>dfs.datanode.data.dir</name>
-         <value>file:///opt/amazon/hadoop/hdfs/datanode</value>
-         <description>Comma separated list of paths on the local filesystem of a DataNode where it should store its\
+    <property>
+        <name>dfs.datanode.data.dir</name>
+        <value>file:///opt/amazon/hadoop/hdfs/datanode</value>
+        <description>Comma separated list of paths on the local filesystem of a DataNode where it should store its\
   blocks.</description>
-     </property>
+    </property>
 
-     <property>
-         <name>dfs.namenode.name.dir</name>
-         <value>file:///opt/amazon/hadoop/hdfs/namenode</value>
-         <description>Path on the local filesystem where the NameNode stores the namespace and transaction logs per\
+    <property>
+        <name>dfs.namenode.name.dir</name>
+        <value>file:///opt/amazon/hadoop/hdfs/namenode</value>
+        <description>Path on the local filesystem where the NameNode stores the namespace and transaction logs per\
  sistently.</description>
-     </property>
+    </property>
+
+	<property>
```

Comment:
Could you add a comment above this property saying why it's being added and what it fixes, or other references?

Same for the other property.

Reply:
Yes added the reference. It's an barely seen issue that we only see in small regions and according to the reference I read. It should be caused by network issue. However, since I'm not able to reproduce the issue, I cannot 100% say that it's the fix. I don't know if it's going to cause large issues because these properties only applies when datanode failed.